packages/tasks/src/hardware.ts

```diff
@@ -0,0 +1,305 @@
+				tflops: 47.87,
+				memory: [192],
+			},
+			MI250: {
+				tflops: 45.26,
+				memory: [128],
+			},
+			MI210: {
+				tflops: 22.63,
+				memory: [64],
+			},
+			"RX 7900 XTX": {
+				tflops: 61.39,
+				memory: [24],
+			},
+			"RX 7900 XT": {
+				tflops: 51.48,
+				memory: [20],
+			},
+			"RX 7900 GRE": {
+				tflops: 45.98,
+				memory: [16],
+			},
+			"RX 7800 XT": {
+				tflops: 7.57,
+				memory: [16],
+			},
+			"RX 7700 XT": {
+				tflops: 7.57,
+				memory: [12],
+			},
+			"RX 7600 XT": {
+				tflops: 7.57,
+				memory: [16, 8],
+			},
+		},
+	},
+	CPU: {
+		Intel: {
+			"Xeon 4th Generation (Sapphire Rapids)": {
+				tflops: 3.8,
+			},
+			"Xeon 3th Generation (Ice Lake)": {
+				tflops: 3.6,
+			},
+			"Xeon 2th Generation (Cascade Lake)": {
+				tflops: 3,
+			},
+			"Intel Core 13th Generation (i9)": {
+				tflops: 3.8,
+			},
+			"Intel Core 13th Generation (i7)": {
+				tflops: 3.6,
+			},
+			"Intel Core 13th Generation (i5)": {
+				tflops: 3,
+			},
+			"Intel Core 13th Generation (i3)": {
+				tflops: 2,
+			},
+			"Intel Core 12th Generation (i9)": {
+				tflops: 2,
+			},
+			"Intel Core 12th Generation (i7)": {
+				tflops: 2,
+			},
+			"Intel Core 12th Generation (i5)": {
+				tflops: 2,
+			},
+			"Intel Core 12th Generation (i3)": {
+				tflops: 1,
+			},
+			"Intel Core 11th Generation (i9)": {
+				tflops: 1,
+			},
+			"Intel Core 11th Generation (i7)": {
+				tflops: 0.76,
+			},
+			"Intel Core 11th Generation (i5)": {
+				tflops: 0.76,
+			},
+			"Intel Core 11th Generation (i3)": {
+				tflops: 0.76,
+			},
+			"Intel Core 10th Generation (i9)": {
+				tflops: 0.76,
+			},
+			"Intel Core 10th Generation (i7)": {
+				tflops: 0.76,
+			},
+			"Intel Core 10th Generation (i5)": {
+				tflops: 0.76,
+			},
+			"Intel Core 10th Generation (i3)": {
+				tflops: 0.76,
+			},
+		},
+		AMD: {
+			"EPYC 4th Generation (Genoa)": {
+				tflops: 4,
+			},
+			"EPYC 3th Generation (Milan)": {
+				tflops: 3,
+			},
+			"EPYC 2th Generation (Rome)": {
+				tflops: 2.9,
+			},
+			"EPYC 1st Generation (Naples)": {
+				tflops: 2.3,
+			},
+			"Ryzen Zen4 7000 (Ryzen 9)": {
+				tflops: 2.3,
+			},
+			"Ryzen Zen4 7000 (Ryzen 7)": {
+				tflops: 1.33,
+			},
+			"Ryzen Zen4 7000 (Ryzen 5)": {
+				tflops: 1.33,
+			},
+			"Ryzen Zen3 5000 (Ryzen 9)": {
+				tflops: 1.33,
+			},
+			"Ryzen Zen3 5000 (Ryzen 7)": {
+				tflops: 1.33,
+			},
+			"Ryzen Zen3 5000 (Ryzen 5)": {
+				tflops: 0.72,
+			},
+			"Ryzen Zen 2  3000 (Threadripper)": {
+				tflops: 0.72,
+			},
+			"Ryzen Zen 2  3000 (Ryzen 9)": {
+				tflops: 0.72,
+			},
+			"Ryzen Zen 2  3000 (Ryzen 7)": {
+				tflops: 0.72,
+			},
+			"Ryzen Zen 2  3000 (Ryzen 5)": {
+				tflops: 0.72,
+			},
+			"Ryzen Zen 2  3000 (Ryzen 3)": {
+				tflops: 0.72,
+			},
+		},
+	},
+	"Apple Silicon": {
+		"-": {
+			"Apple M1": {
+				tflops: 2.6,
+				memory: [16, 24, 32, 64, 96, 128],
+			},
+			"Apple M1 Pro": {
+				tflops: 5.2,
+				memory: [16, 24, 32, 64, 96, 128],
```

Comment:
```suggestion
				memory: [16, 24, 32],
```

Not sure ever was a 24 GB configuration, but max was announced to be 32: https://www.apple.com/hu/newsroom/2021/10/introducing-m1-pro-and-m1-max-the-most-powerful-chips-apple-has-ever-built/